### PR TITLE
Add prow/cluster/monitoring/README.md

### PR DESCRIPTION
**Pull request by hongkailiu**
https://github.com/kubernetes/test-infra/issues/12854

### CODE REVIEW

File: prow/monitoring/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+The `svc` should be available on prometheus web UI: `Status` &rarr; `Targets`.
+
+_Note_ that the serviemonitor has to have label `app` as key (value could be an arbitrary string).
+
+### Add a new grafana dashboard
+
+We use [jsonnet](https://jsonnet.org) to generate the json files for grafana dashboards and [jsonnet-bundler](https://github.com/jsonnet-bundler/jsonnet-bundler) to manage the jsonnet libs.
+Developing a new dashboard can be achieved by
+
+* Create a new file `<dashhoard_name>.jsonnet` in folder `grafana_dashboards`.
```
Comment (stevekuznetsov) [inline prow/monitoring/README.md]: Add a relative link to the dir here

File: prow/monitoring/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* Use `bazel` to build the corresponding json file `<dashhoard_name>.json` in folder `dashboards_out`.
```
Comment (stevekuznetsov) [inline prow/monitoring/README.md]: Leave a copy-pastable command to run

File: prow/monitoring/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+This folder contains the manifest files for monitoring prow resources.
+
+## Deploy
+
+TODO: The deployment has been integrated into our CI system. 
+
+A successful deploy will spawn a stack of monitoring for prow in namespace `prow-monitoring`: _prometheus_, _alertmanager_, and _grafana_.
+
+## Add more dashboards
```
Comment (stevekuznetsov) [inline prow/monitoring/README.md]: The sub-headings (`###`) are useful but this one I am not sure adds any value

File: prow/monitoring/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+Suppose that there is an App running as a pod that exposes Prometheus metrics on port `n` and we want to include it into our prow-monitoring stack.
+First step is to create a k8s-service to proxy port `n` if you have not done it yet.
+
+### Add the service as target in Prometheus
```
Comment (stevekuznetsov) [inline prow/monitoring/README.md]: Let's add a copy-pastable example YAML

File: prow/monitoring/README.md (hunk 1 above)
Comment (hongkailiu) [reply prow/monitoring/README.md]: done.

File: prow/monitoring/README.md (hunk 2 above)
Comment (hongkailiu) [reply prow/monitoring/README.md]: done

File: prow/monitoring/README.md (hunk 3 above)
Comment (hongkailiu) [reply prow/monitoring/README.md]: Removed.

File: prow/monitoring/README.md (hunk 4 above)
Comment (hongkailiu) [reply prow/monitoring/README.md]: done

File: prow/monitoring/README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+      - default
+  selector:
+    matchLabels:
+      app: ghproxy
+
+```
+
+The `svc` should be available on prometheus web UI: `Status` &rarr; `Targets`.
+
+_Note_ that the `serviemonitor` has to have label `app` as key (value could be an arbitrary string).
```
Comment (stevekuznetsov) [inline prow/monitoring/README.md]: typo here on `serviemonitor`

**Comment by hongkailiu** [comment]
/hold 
https://github.com/kubernetes/test-infra/pull/13401

**Pull request merged**